.github/ISSUE_TEMPLATE/FeatureRequest.yml

```diff
@@ -0,0 +1,43 @@
+name: 💡 Feature request
+description: Have a new idea/feature ? Please suggest!
```

Comment:
Remove space before the question mark. Besides, there should be an object (`it`) after `suggest`.